src/bin/zmlocalconfig

```diff
@@ -16,7 +16,7 @@ umask 0027
 
 java="${ROOT}/common/bin/java"
 
-CP="${ROOT}/lib/jars/*"
+CP="${ROOT}/mailbox/jars/*"
```

Comment:
So you want to create another folder in /opt/zextras ? personally I would avoid it. We already have /opt/mailboxd. I would put jars in any existent `lib/jars` folder

Reply:
Resolving as we discussed it's ok to use /opt/zextras/mailbox